src/main/kotlin/com/sourcegraph/common/UpgradeToCodyProNotification.kt

```diff
@@ -23,7 +23,7 @@ private constructor(title: String, content: String, shouldShowUpgradeOption: Boo
   init {
     icon = Icons.CodyLogo
     val learnMoreAction: AnAction =
-        object : DumbAwareAction("Learn more") {
+        object : DumbAwareBGTAction("Learn More") {
```

Comment:
no strong feelings but `Learn more` looks better imho

Reply:
That is against IntelliJ action names guidelines, it's highlighted in the IDE ;)